warehouse/core/src/main/java/datawave/ingest/mapreduce/handler/dateindex/DateIndexUtil.java

```diff
@@ -21,6 +21,7 @@ public class DateIndexUtil {
      * Format the date into yyyyMMdd
      * 
      * @param date
+     *            then date to be formatted
```

Comment:
the date

Reply:
thanks. Still have a few more annotations to go but I'll make this adjustment as well.